flink-mesos/src/main/java/org/apache/flink/mesos/util/MesosArtifactServer.java

```diff
@@ -135,7 +134,8 @@ protected void initChannel(SocketChannel ch) {
 
 				// SSL should be the first handler in the pipeline
 				if (sslFactory != null) {
-					ch.pipeline().addLast("ssl", sslFactory.createNettySSLHandler());
+					ch.pipeline().addLast("ssl",
```

Comment:
I do not understand this `[FLINK-9816][network] netty-fy SSL configuration` commit. What does it do? There are no new tests, no changes in the existing tests, no documentation and nothing in the commit message :(

Is it refactor? If so please explain in the commit message what are you refactoring and why.